### ISSUE TITLE
Aktoren ausstellen bei nicht aktivem Raum

### ISSUE THREAD

**Issue by bastian1607** [rg-engineering/ioBroker.heatingcontrol#292]
Ich habe den Raum Wintergarten und dort Aktoren (zwei Zigbee-Steckdosen an denen Elektroheizungen hängen). Wenn der Raum aktiv ist und entsprechend Temperaturen eingestellt sind, funktioniert das wunderbar. 
Um das möglich zu machen habe ich ein Fake-Thermostat erstellt und die Aktoren eingebunden.

![image](https://user-images.githubusercontent.com/79959369/149462006-fd91e643-58ad-48e4-a1b9-d874c6265c1d.png)

![image](https://user-images.githubusercontent.com/79959369/149462021-f5f287f8-5f1b-4b5f-ba81-2110fa5e9e90.png)

Nun dachte ich, dass ich mit dem neuen Datenpunkt um den Raum aktiv zu stellen steuern kann ob die Aktoren agieren oder eben nicht. Heißt: brauche ich einen warmen Wintergarten oder eben nicht. 
Wenn ich (über die Vis) den Raum aktiv stelle, stellt sich der Raumstatus auf "Auto" und die Aktoren springen entsprechend an. Soweit, so gut.
Wenn ich jedoch den Raum auf nicht aktiv stelle, ist der Raumstatus auf "no heating", die Aktoren bleiben jedoch an. 

![image](https://user-images.githubusercontent.com/79959369/149462766-10de90ff-7a98-4a8f-9f9b-d0cef3e93600.png)

![image](https://user-images.githubusercontent.com/79959369/149462777-aabec375-a346-4bad-80a6-937a0a5e4851.png)

![image](https://user-images.githubusercontent.com/79959369/149462796-b73929d1-f826-4d71-872a-967c2f5fa866.png)

Änderung:
Wenn der Raum auf "no heating" steht bzw. inaktiv ist, sollten die Aktoren auch mit ausgehen.

**Comment by rg-engineering** [comment]
ist in der 2.8.4 enthalten.
Funktioniert jetzt genau so wie bei Änderung der Heizperiode. Einstellung, auf welchen state der Aktor schalten soll, wird auch davon übernommen

**Comment by bastian1607** [comment]
Hi,
vielen Dank, ich konnte es heute testen. 

Die Aktoren schalten sich jetzt aus. Jetzt ist mir nur noch eine Kleinigkeit aufgefallen:
Als Zieltemperatur bleibt der alte Wert in den Objekten erhalten.
![image](https://user-images.githubusercontent.com/79959369/150740115-b7f2c65a-13ce-4b55-80ae-225d4643e4a9.png)

Bedeutet aber auch, dass in der Übersicht es so aussieht als wenn der Raum angeheizt werden müsste.
![image](https://user-images.githubusercontent.com/79959369/150740201-9179afe1-6d07-461b-b426-51e78e982267.png)

Richtig wäre ja hier, dass er als Zielwert das Objekt heatingcontrol.0.vis.RoomValues.MinimumTemperature setzt, oder?

**Comment by rg-engineering** [comment]
Warum MinimumTemperatur? Das entspricht ja auch nicht der Realität. Ich könnte den Zielwert ausblenden oder auf n/a setzen, wenn Status "no heating" ...

**Comment by bastian1607** [comment]
Richtig, Gedankenfehler meinerseits. Der Raum soll ja in keinem Fall beheizt werden, wenn der Raum nicht aktiv ist. n/a wäre ja folgerichtig und entsprechend der Logik mit nicht vorhandenen externen Sensoren meiner Meinung nach korrekt.

**Comment by rg-engineering** [comment]
![grafik](https://user-images.githubusercontent.com/8238668/151032081-921b44a0-9df9-4bdd-9b31-2e56caf304a9.png)

**Comment by bastian1607** [comment]
Getestet und soweit sehr gut. Vielen Dank!

**Comment by bastian1607** [comment]
Neuer Test:
Grundsätzlich scheint es bei den Aktoren momentan doch nicht zu funktionieren. Bei nicht aktivem Raum setzt das Skript trotzdem die Aktoren auf "ein". 
![image](https://user-images.githubusercontent.com/79959369/151757296-649bf369-91e7-4429-8514-279fd17a5206.png)

Meine Befürchtung ist, dass er die Temperatur aus dem Profil (CurrentTarget) nimmt und damit rechnet. 
![image](https://user-images.githubusercontent.com/79959369/151757392-72d6639d-5477-4e24-93fd-3ceeada26f37.png)


Ich habe einen zweiten Raum getestet mit Thermostaten. Dort ist das ebenso zu finden. Der Raum ist nicht aktiv, trotzdem werden die Thermostate mit der TargetTemperatur geschaltet, bzw. zumindest keine neue Temperatur übermittelt. 
![image](https://user-images.githubusercontent.com/79959369/151757727-59b200f1-fd61-43c4-9332-2a3e5f326ab6.png)

![image](https://user-images.githubusercontent.com/79959369/151757772-e7a064e9-5016-4885-b9b8-89731b533d77.png)

Für mich wäre hier richtig, dass bei nicht aktivem Raum die letzte Temperatur die MinimumTemperatur des Raumes ist bzw. als TargetTemperatur gesetzt wird solange bis der Raum wieder aktiv gesetzt wird.

**Comment by rg-engineering** [comment]
In einem nicht aktiven Raum bleibt die Zieltemperatur einfach auf dem letzten Stand stehen. Es werden keine Profiländerungen vorgenommen. Es gibt sicher user, die das Absenken auf Minimal-Temperatur nicht gut finden würden.... 
Außerdem: wenn ich auf MinimalTemperatur absenke, müsste ich auch noch überwachen, dass keiner am Thermostat selbst dreht. und was mache ich dann?
ABER: ich habe ja eingebaut, dass bei nicht aktiven Raum die Aktoren einen definierten Zustand einnehmen. Deshalb stimme ich zu, dass im ersten Bild der Aktor nicht "ein" sein sollte. Das ist ein Fehler, den ich beheben würde. Leider kann ich das hier nicht nachvollziehen. Hast du dazu evtl. einen debug-log?
